src/app/api/feedback/route.js

```diff
@@ -0,0 +1,21 @@
+import { NextResponse } from 'next/server';
+import { prisma } from '@/lib/db';
+import SendFeedback from '@/lib/validationSchemas/sendFeedbackSchema';
+
+export async function POST(request) {
+  try {
+    const feedbackData = await request.json();
```

Comment:
Let's write a validation function(e.g. called validateBody) instead of hanlding a zod  result each time by hand.